tests/pytests/test_cursors.py

```diff
@@ -246,3 +246,47 @@ def testExceedCursorCapacity(env):
 def testExceedCursorCapacityBG():
     env = Env(moduleArgs='WORKER_THREADS 1 MT_MODE MT_MODE_FULL')
     testExceedCursorCapacity(env)
+
+def testCursorOnCoordinator(env):
```

Comment:
Maybe we can think of how to test cases where the cursor is timeout on one shard but not on the other? This could have happened even before this change but now it is more likely to happened.

Reply:
I'll add a `TODO` regarding that, let's wait for when we can:
1. pass `MAXIDLE` to the shards (currently using the default of 5 minutes)
2. Tell if a timeout occurred

Reply:
I guess we will need RediSearch mock to control the RediSearch reply to the coordinator and verify that the coordinator act as expected. Maybe we should open a task for this?

Reply:
@DvirDukhan @oshadmi